scripts/tuple_tree_generator/tuple_tree_generator/templates/struct.h.tpl

```diff
@@ -148,6 +148,12 @@ public:
     }
   /** endif **/
 
+  /** if struct.inherits **/
+  /// Parent constructor
+  /*= struct.name =*/(const /*= struct.inherits.name =*/ &Parent)
+    : /*=- struct.inherits.name =*/(Parent) {}
+  /** endif **/
```

Comment:
Why is this needed?

Reply:
Ok, I tried and everything build without it. So it must be a leftover from a previous iteration or something.

Thanks for catching it!

Reply:
Dropped.